tensorboard/uploader/proto/experiment.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package tensorboard.service;
+
+import "google/protobuf/timestamp.proto";
+
+// Metadata about an experiment.
```

Comment:
It's good form to indicate (once we support updating some of this metadata) which fields are output-only and cannot be directly set by the client.  (In this case, all of them except name and description.)

See e.g. https://aip.dev/203#output-only (I don't think we necessarily need to use the programmatic specifier, but having the documentation in a comment is still valuable)